apps/studio/src/features/editing-experience/components/LayoutSelection.tsx

```diff
@@ -102,59 +92,45 @@ function LayoutSelection(props: LayoutSelectionProps): JSX.Element {
                     key={layoutEntry.layoutTypename}
                     p="0"
                     gap="0"
-                    cursor="pointer"
                     onClick={() => {
                       setSelectedLayout(layoutEntry)
                     }}
                     role="group"
+                    as="button"
                   >
                     <Box
-                      className="hover-border"
+                      // className="hover-border"
                       borderRadius="8px"
                       border="2px"
                       p="1.5rem 1.5rem 0 1.5rem"
                       mb="1.25rem"
                       position="relative"
-                      borderColor={
-                        selectedLayout.layoutTypename ===
-                        layoutEntry.layoutTypename
-                          ? 'base.divider.brand'
-                          : 'base.divider.medium'
-                      }
-                      bgColor={
+                      borderColor="base.divider.medium"
+                      bgColor="interaction.muted.main.hover"
+                      data-active={
                         selectedLayout.layoutTypename ===
                         layoutEntry.layoutTypename
-                          ? 'interaction.muted.main.active'
-                          : 'interaction.muted.main.hover'
-                      }
-                      sx={
-                        selectedLayout.layoutTypename !==
-                        layoutEntry.layoutTypename
-                          ? {
-                              transition:
-                                'border-color 300ms ease-out, opacity 300ms ease-out, background-color 300ms ease-out',
-                            }
-                          : {}
-                      }
-                      _groupHover={
-                        selectedLayout.layoutTypename !==
-                        layoutEntry.layoutTypename
-                          ? {
-                              borderColor: useToken(
-                                'colors',
-                                'interaction.main-subtle.hover',
-                              ),
-                              bgColor: useToken(
-                                'colors',
-                                'interaction.main-subtle.default',
-                              ),
-                            }
-                          : {}
-                      }
+                          ? true
+                          : null
+                      } // have to use null so that React will not parse this jsx attribute. having the [data-active] selection attribute will trigger _active.
```

Comment:
ah, I have a handy function to alleviate this, might want to use it here too:

```ts
// utils somewhere
/**
 * Function to return undefined if the value is falsy, otherwise true
 */
export const dataAttr = (value: unknown) => (!!value ? true : undefined);

// Usage:
data-active={dataAttr(selectedLayout.layoutTypename === layoutEntry.layoutTypename)}
```